.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+  // Use IntelliSense to learn about possible Node.js debug attributes.
```

Comment:
back to a computer so I can make an inline comment: while VS Code supports a superset of JSON with comments, not all tools do and we've had problems in the past with that. So let's keep all of our JSON valid, as restrictive as that may be.